airbyte-integrations/connectors/source-bigcommerce/metadata.yaml

```diff
@@ -35,6 +35,10 @@ data:
   # This connector is not enabled for OSS/Cloud
   #
   # connectorTestSuitesOptions:
+  #   - suite: liveTests
```

Comment:
Intentionally keeping this commented for now?

Reply:
Yes, the sandbox connection isn't passing `check` and it has low usage.